.env-template

```diff
@@ -1,4 +1,12 @@
 API_REF=
-MONGO_URI=
 API_PORT=
+
+MONGO_URI=
+MONGO_ROOT_USERNAME=
+MONGO_ROOT_PASSWORD=
+# MONGO_DATABASE is only used in docker-compose.
```

Comment:
these type of comments should be in our project README imo, under a section about environment variables. Thoughts?

Reply:
Agreed. Will make changes. Makes more sense that way.